controllers/getQuestion.js

```diff
@@ -0,0 +1,17 @@
+const models = require('../models/index');
+const getQuestion = async (req, res) => {
+    try {
+        const userId = parseInt(req.params.user_id);
+        const questions = await models.questions.findAll({
+            where: { user_id: userId },
+        });
+        return res.status(200).json({ success: 'true', data: questions });
```

Comment:
- Since you are fetching more than one question, then rename the function to getQuestions.
- Since you are going to remove the URL parameter mentioned on the comment for the route file then modify `const userId = parseInt(req.params.user_id);` to `const userId = parseInt(req.query.user_id);`
- Change line 5-7 to like below
```javascript
const whereClause = {};
if(userId)
whereClause.user_id= userId;
const questions = await models.questions.findAll({
           raw: true,
            where: whereClause,
        });
```
So this way, if the userId is supplied, then questions for that  user is returned else all the questions on the system are  returned. 

**Research on how to paginate data with Sequelize so you don't have to send millions of answers to the user at once**